11ty/definitions/adhd.md

```diff
@@ -0,0 +1,30 @@
+---
+title: ADD/ADHD
+slug: adhd
+defined: true
+speech: noun
+reading:
+- text: "What Is ADHD? Meaning, Symptoms & Tests"
+  href: https://www.additudemag.com/what-is-adhd-symptoms-causes-treatments/
+- text: "No Time To Confront Racism In Neurodiversity"
+  href: https://blackgirllostkeys.com/adhd/no-time-to-confront-racism-in-neurodiversity/
+- text: DSM-5 ADHD Diagnosis
+  href: https://www.cdc.gov/ncbddd/adhd/diagnosis.html
+
+---
+
+ADHD is a neurological disorder that affects someone's ability to focus or execute tasks. It is commonly thought of as "the hyper kid in class", though it is much more complex and nuanced.
+
+## Common Usage
+
+People often say "I'm so ADD/ADHD" when they're referring to short periods of hyperactivity or inability to focus. It's used colloquially much like "I'm so OCD," and has many of the same negative ramifications for neurodivergent folks.
+
+When people think of ADHD, they think of a very specific character. This character is almost always a boy, and he is usually hyperactive and extremely distractible.
```

Comment:
```suggestion
When people think of ADHD, they think of a very specific character. This character is almost always an able-bodied white boy, and he is usually hyperactive and extremely distractible.
```

Reply:
I might suggest here the role of media: How in TV/movies/books, ADHD characters are almost always played by white boys.